core/server/api/canary/utils/validators/input/schemas/members.json

```diff
@@ -34,6 +34,9 @@
                 "id": {
                     "strip": true
                 },
+                "labels": {
+                    "$ref": "#/definitions/member-labels"
```

Comment:
Why wasn't it possible to use `"$ref": "members#/definitions/member-labels"` like in `members-edit` schema?

Reply:
The ref definitions here seem to be relative to the file, and in this case the definition of `member-labels` is defined in `members.json` file itself, so its reference starts relative to it unlike `members-edit` schema where it has to know to pick this in `members` file. Think I went with this first before it failed, and we are also using the same pattern for other ref definitions in posts/tags like [here](https://github.com/TryGhost/Ghost/blob/master/core/server/api/canary/utils/validators/input/schemas/posts.json#L119), so switched this as well :)